### PR TITLE
Djangobower

### PR DESCRIPTION
**Pull request by benrudolph**
the ansible component for djangobower: https://github.com/dimagi/commcare-hq/pull/8364

### CODE REVIEW

File: ansible/roles/common_installs/tasks/main.yml [hunk 1]
```diff
@@ -32,7 +36,7 @@
     - htop
     - tmpreaper
     - mailutils
-    - npm
+    - nodejs
```
Comment (benrudolph) [inline ansible/roles/common_installs/tasks/main.yml]: this caused me quite a bit of pain. we have different installs of node on different machines which made the path to `bower` different. i think this should be the right way; i know we've gone back and forth on nodejs and npm many times. with this ppa nodejs should install npm as well. see: http://stackoverflow.com/a/16303380/835696

@dannyroberts @TylerSheffels

Comment (TylerSheffels) [inline ansible/roles/common_installs/tasks/main.yml]: I agree, this always seems to be a problem. is it worth specifying the version here e.g. `- nodejs=0.10.37` or whatever the exact version is?

Comment (benrudolph) [inline ansible/roles/common_installs/tasks/main.yml]: we specify it when we use `n` to upgrade:
```
 - name: Update Node
   command: 'n 0.10.29'
```

**Comment by benrudolph** [comment]
@proteusvacuum @sravfeyn this is g2g

**Pull request merged**